src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Rectangle implements Figure {
+    private String color;
+    private double height;
+    private double width;
+
+    public Rectangle(String color, double height, double width) {
+        this.color = color;
+        this.height = height;
+        this.width = width;
```

Comment:
Creating a `Rectangle`, avoid passing expressions directly in the constructor. Instead, create separate variables and pass them on for better code readability.